digital_land/phase/convert.py

```diff
@@ -114,6 +115,28 @@ def convert_features_to_csv(input_path, output_path=None):
     return output_path
 
 
+def convert_json_to_csv(input_path, output_path=None):
+    with open(input_path, "r") as json:
+        js = json_stream.load(json)
+        columns = None
+
+        for item in js.items():
+            if item[0] in ["columns"]:
+                columns = [x for x in item[1].persistent()]
+
+            if item[0] in ["data"] and columns is not None:
+                with open(output_path, "w") as csv_file:
+                    cw = csv.writer(csv_file)
+                    cw.writerow(columns)
+
+                    for row in item[1]:
+                        cw.writerow(row)
```

Comment:
Does this work if data is before columns in the file?

Reply:
I suspect not, but if it's not we can't process it as a stream.

Reply:
Updated to handle the data being first. It won't stream it in this case, but it will handle it.